.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: "Update aHomebrew formula under alexellis name with latest release"
+on:
+  push:
+    branches:
+      - master
+      - auto-update-release
+    paths:
+      - "*"
+      - "!README.md"
+  schedule:
+    # Run workflow once a day to grab latest release
+    - cron: "3 0 * * *"
+
+jobs:
+  brew:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
+      - name: Render latest template
+        run: |
+          ruby render.rb k3sup | tee Formula/k3sup.rb
+          ruby render.rb arkade | tee Formula/arkade.rb
+          
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
I am worried that we are potentially sending my personal access token to this unknown GitHub action by a third-party.

What if it gets compromised? Someone could collect all the tokens, correct?

Is there not a standard `actions/commit` etc? (assuming that `actions` is owned by GitHub)

@LucasRoesler, have you looked into this much?

Reply:
https://humanwhocodes.com/blog/2020/07/safely-use-github-actions-in-organizations/

Reply:
Can we fork the action? they are basically gitrepos and a docker image, so we could just fork and track the upstream to monitor it

Reply:
That action is just a node:12 image and the script, it would not be too difficult to fork and track https://github.com/stefanzweifel/git-auto-commit-action

Reply:
Or we could use git without the action and for the github token permission they can be edited https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token

Reply:
Isn't there a standard commit action from the GitHub team? Why do we need stefanzweifel's one?

Reply:
there is no standard commit action from the GitHub team and this was not copied and pasted 😞  from  Snyk's repo because in  Snyk's repo they used `git` directly and the reason that I used stefanzweifel's action over standard `git` for readability.
but think if I use `git` command directly that would be ok right? 🤔

Reply:
Yes, please can you do that :)

Reply:
@amalkh5 It looks like you have not decided to continue with this PR.

@utsavanand2 please can you take this over in a new PR with only standard GH actions?